packages/cacti-weaver-driver-fabric/package.json

```diff
@@ -0,0 +1,81 @@
+{
+    "name": "@hyperledger/cacti-weaver-driver-fabric",
```

Comment:
If this package is meant to serve similar purpose as fabric connector, I'd propose to name it according to cacti standards (i.e. something like `cactus-plugin-ledger-connector-weaver-fabric` or `cactus-plugin-ledger-connector-fabric-weaver`)

Reply:
@outSH The names for several packages contain the `cactus-` prefix because they were carried over from the Cactus project into Cacti when the merger with Weaver occurred. Our intention, going forward, was to use `cacti` instead of `cactus` (or `weaver`). The reason we are still using `cactus` and `weaver` in several package names is because the packages of both the contributing repos are still distinct within Cacti.

We are aiming for a gradual integration whereby we eliminate redundant packages. This PR is simply a stepping stone to that whereby we expose the Weaver packages through the standard Cacti monorepo. After this, we will expose all other Weaver packages the same way. Later we will combine, merge, and eliminate various packages within the `packages` folder. For example, `cacti-weaver-driver-fabric` and `cactus-plugin-ledger-connector-fabric` are meant to serve basically the same purpose, so we ought to merge their functions and APIs into a single module called, let's say, `cacti-plugin-ledger-connector-fabric`. But this kind of refactoring (actually, it's more than that) will take time and effort.

So, to cut to the chase, any new package we create within this repository I believe ought to have the `cacti-` prefix. The suffix is something we can debate.

For your reference, see the roadmap and target architecture sketches in https://github.com/hyperledger/cacti/blob/main/ROADMAP.md#cacti-v2. (I know we've long passed the dates indicated there, so we will need to update that page with a more realistic estimate.)

@petermetz Any thoughts or suggestions on this?

Reply:
> We are aiming for a gradual integration whereby we eliminate redundant packages. This PR is simply a stepping stone to that whereby we expose the Weaver packages through the standard Cacti monorepo. After this, we will expose all other Weaver packages the same way. Later we will combine, merge, and eliminate various packages within the `packages` folder. For example, `cacti-weaver-driver-fabric` and `cactus-plugin-ledger-connector-fabric` are meant to serve basically the same purpose, so we ought to merge their functions and APIs into a single module called, let's say, `cacti-plugin-ledger-connector-fabric`. But this kind of refactoring (actually, it's more than that) will take time and effort.

@VRamakrishna That was clear to me, my intention was to keep standardized naming convention for temporary weaver packages as well, so that they can be easily identified and managed. I'm don't have strong opinions on the exact naming. 
As for `cacti` prefix, some scripts still have (or at least had a while ago) hard-coded `cactus` for identifying packages, that's why I keep using `cactus`. I kinda assumed we will do the project-wide switch to `cacti` in a single commit at some point, but if it works already then that's even better.

Reply:
@outSH @VRamakrishna Sorry for being late to the party! I could argue for and against all of these and also agree that it's not a big deal either way if the pacakge is temporary. So if it is temporary, I'm good with the pkg name as is. If it will stick around I'll also suggest to call it something like `cacti-plugin-ledger-connector-fabric-weaver` but it's really hard to pin down which one makes most sense because it's (for now) not an implementation of the connector plugin interface. So we are trying to name something that we already know will change over time and so it could stand to reason either way, as in, the current state of it could inform the name to be less confusing now and more confusing later, or the future state of it could inform the name to be more confusing now and less later... :-)

1. I agree that for the new packages `cacti` is a better prefix since the end goal is to have all of them end up that way.
2. I also agree that the hierarchical & flattened naming structure we have in place helps a lot (at least to me) to find the code I'm looking for and should be kept where possible (even if it feels annoyingly verbose at times)
3. Also agree that we need to do the "big rename" rather sooner than later and definitely before the 2.0.0 release goes out because we get to make breaking changes prior to that date (this is my usual reminder that now is the "easy" time when we can do things like that - next time will be right before 3.0.0)

With all that said, I think I'm OK with the name for now as-is and we should rename it later, potentially in v3.0.0.

What I'll nit-pick on is much worse... :D I'd love it if we called the `readme.md` file `README.md` just keep with the conventions.